default/scripting/ship_parts/Hangars.focs.txt

```diff
@@ -11,11 +11,11 @@ Part
     icon = "icons/ship_parts/fighters-1.png"
 
 Part
-    name = "FT_HANGAR_1"
+    name = "FT_HANGAR_2"
```

Comment:
I think three different hangars, just trading off capacity vs. fighter damage, is a bit much to unlock all at once.